fava_investor/common/beancountinvestorapi.py

```diff
@@ -46,9 +46,7 @@ def get_account_open_close(self):
         return getters.get_account_open_close(self.entries)
 
     def get_account_open(self):
-        oc = getters.get_account_open_close(self.entries)
-        opens = [e for e in oc if isinstance(e, Open)]
-        return opens
+        return [e for e in self.ledger.entries if isinstance(e, Open)]
```

Comment:
getters is a higher level function to get close/opens, so I'd prefer to retain that here and in favainvestorapi.py

Reply:
I think I addressed your comments.